Resources/Prototypes/Entities/Objects/Weapons/Guns/Launchers/launchers.yml

```
@@ -85,7 +85,41 @@
     steps: 2
     zeroVisible: false
   - type: Appearance
-
+  
+- type: entity
+  name: singularity buster rocket launcher
+  parent: [BaseWeaponLauncher, BaseGunWieldable, BaseGrandTheftContraband]
+  id: WeaponLauncherSingularityBuster
+  description: An experimental rocket launcher designed to fire rockets capable of decomposing a singularity. 
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Launchers/buster_rocket.rsi
+    layers:
+      - state: base
+        map: ["enum.GunVisualLayers.Base"]
+  - type: Clothing
+    sprite: Objects/Weapons/Guns/Launchers/buster_rocket.rsi
+  - type: Tag
+    tags:
+      - HighRiskItem
```

Tags are indented wrong, there's more indentations wrong, this is just an example
```suggestion
  - type: Tag
    tags:
    - HighRiskItem
```